### DISCUSSION TITLE
Questions & brainstorming around PyLadies Conference

### DISCUSSION THREAD

**Discussion by econchick** [pyladies/global-organizing#139] [Ideas]
[Pyladies Conf Proposal](https://github.com/pyladies/global-organizing/files/11436469/Pyladies.Conf.Proposal.pdf) for review.

During the most recent Global Council meeting, we're all in agreement that it should happen. Now, the next steps are to collect questions about the conference. Please add them here!

**Comment by econchick** [comment]
When do we want to have this conference? If no one has ideal dates in mind, let's just maybe select a month, or at least factors that should go into this decision.

I'm not sure if there's a maintained calendar of all Python-related conferences world-wide, but if there is, it'd be good to try our best not to collide with any of them.

**Reply by Mariatta** [reply]
https://github.com/python-organizers/conferences maintains a list of known Python conferences.

**Reply by econchick** [reply]
Thanks Mariatta! this is great

**Comment by econchick** [comment]
Who are the core folks interested in organizing the conference?

**Reply by whykay** [reply]
It depends when and where, and I also want to help where I can! 💪

**Reply by econchick** [reply]
You're awesome @whykay ! 

I definitely want to help, too!

**Reply by DennyPerez18** [reply]
Thank you!! 
Amazing!! So excited. It will be such a thrill to see this event become a reality! 🥰

**Comment by DennyPerez18** [comment]
Hey @econchick 
**1-** The estimated date could be set for the end of November. But it is only a suggestion; we can see this in our first meeting.

**2-** The team I was discussing the idea of organizing the conference is the Chapter Pyladies En Español, organized by Maria José, Johana, Elka and Cristián Maureira, the organizer of PyLadies Berlin.
They are fully available to collaborate in the organization of the event.
@Mariatta is also already helping with the planning.

I suggest we create a form to call for volunteers on the Pyladies Slack platform, which will facilitate the recruitment process and ensure effective communication among team members.

**Comment by econchick** [comment]
A sub-thread about the online streaming platform:

I know Hubilo is being considered, and may have some cost issues. I just wanted to suggest a cheap alternative in case it’s too prohibitive: 

PyGotham (NYC-region based PyCon) has been doing online conferences since COVID, and uses Twitch -> Youtube (private stream access during the conference, then the videos are posted later on youtube for everyone). They had the speakers pre-record their talks, with an AV person managing the stream to Twitch ( @elainewong actually did this for us, PyLadies leader from Toronto). 

I think also had live introductions from organizers/volunteers. And the managing between pre-recorded streams and live streams was all done thru [OBS](https://obsproject.com/) (free/open source). 

The social aspect was then managed on Discord, which may not give the same level of social experience that Hubilo does, but it’s familiar to a lot of folks.

Just a thought

**Reply by DennyPerez18** [reply]
Yes! Excellent options. 💪

We might as well consider one of those options to keep costs down.
We suggested Hubilo to have less work and effort in terms of logistics and organization and keep the conference centralized with everything happening in one place, keeping in mind the magnitude of the conference and the structure we want it to handle is:
Different time zones, multiple tracks, multiple languages, pre-recorded talks and live talks.
We also love the way we network and socialize through the platform.

Today at the meeting, Let's see how expensive Hubilo can be. 🙏

🥰 Yeiiii!!!! I had the opportunity to work with @elainewong organizing a conference (OctogatosConf) where she did an excellent job managing the AV stream.

**Reply by DennyPerez18** [reply]
With a Streamyard account to handle broadcasts, Twitch and Youtube for Streaming and Discord for social, it's also a good alternative. 🤔. It's the way we do for PyCon Chile and Python Brasil.

**Reply by Mariatta** [reply]
I would strongly suggest we use Hubilo platform and focus our energy on getting the necessary fund/sponsorship for it.

Yes, alternative ways like Discord + streamyard + youtube + whatever exists, but at the cost of time and effort from volunteers/organizers to coordinate and manage the various platforms. As attendee the experience of using various tools can feel confusing and not ideal. Having platform that is easy to use will make the event more welcoming and less overwhelming to beginners. Let's invest in the right tooling and provide great online experience to the attendees so that they will want to re-join in the future.
